src/components/Dashboard.vue

```diff
@@ -635,6 +636,18 @@ export default Vue.extend({
     naIfEmptyStringArray(arr: readonly string[]) {
       return arr && arr.length !== 0 && arr[0] !== '' ? arr : ['N/A'];
     },
+
+    deleteCourseFromSemesters(subject: string, number: string, uniqueID: number) {
+      const updatedSemesters = this.semesters.map(semester => {
+        const coursesWithoutDeleted = semester.courses.filter(course => {
+          return (
+            course.subject !== subject || course.number !== number || course.uniqueID !== uniqueID
```

Comment:
Can we just check course.uniqueID here instead of subject and number as well? I think that is the standard across the codebase, and you would not have to emit and pass through as many props

Reply:
That sounds good